Formula/tfsec.rb

```diff
@@ -0,0 +1,17 @@
+class Tfsec < Formula
+  desc "Static analysis powered security scanner for your terraform code"
+  homepage "https://github.com/tfsec/tfsec"
+  url "https://github.com/tfsec/tfsec/archive/v0.30.1.tar.gz"
+  sha256 "95c0b856c95185315cdad4b8442b65cd8178664ed36d776813230d2fb15c43ac"
+  license "MIT"
+  depends_on "go" => :build
+
+  def install
+    system "scripts/install.sh", "v0.30.1"
```

Comment:
Does this download anything?

Reply:
Does it?

Reply:
Sorry, I typed then resolved without commenting. 

No, it does an in place build using `go build` with
```
env GO111MODULE=on CGO_ENABLED=0 go build -ldflags "-X github.com/tfsec/tfsec/version.Version=${1}" ./cmd/tfsec
```